examples/variational_autoencoder.py

```diff
@@ -40,8 +37,8 @@ def sampling(args):
 x_decoded_mean = decoder_mean(h_decoded)
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
-    kl_loss = - 0.5 * K.mean(1 + z_log_std - K.square(z_mean) - K.exp(z_log_std), axis=-1)
+    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
@RuiShu Now I see the point, thank you. And the multiplication makes sense then, cool.